### PR TITLE
Auto-detect `ethereum-network`

### PR DESCRIPTION
**Pull request by tilacog**
- Rely on chainID from ethereum provider instead of `argv.ethereumNetwork`.
- Remove `--ethereum-network` from `indexer-agent`.

### CODE REVIEW

File: packages/indexer-common/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- The `ethereum-network` is now inferred prom provider's `chainId`
```
Comment (fordN) [inline packages/indexer-common/CHANGELOG.md:9]: Small typo here: "prom" --> "from"

File: packages/indexer-common/src/indexer-management/types.ts [hunk 2]
```diff
@@ -149,16 +149,31 @@ export interface NetworkEpoch {
...
-    return Object.keys(CAIPIds).filter((name) => CAIPIds[name] == identifier)[0]
-  } catch (error) {
-    throw new Error(`Failed to match chain ids to a network alias`)
+const Caip2ByChainId: { [key: number]: string } = {
+  1: 'eip155:1',
+  5: 'eip155:5',
+  100: 'eip155:100',
+}
+
+/// Unified entryfpoint to resolve CAIP ID based either on chain aliases (strings)
```
Comment (fordN) [inline packages/indexer-common/src/indexer-management/types.ts:164]: small typo: entryfpoint --> entrypoint

**Review by fordN — APPROVED** [review]
👍

**Comment by tilacog** [comment]
I'll squash some commits before mering

**Pull request merged**